src/Sylius/Bundle/WebBundle/Resources/views/Frontend/Cart/summary.html.twig

```diff
@@ -43,10 +42,11 @@
         {% endfor %}
     </tbody>
     <tfoot>
-        {% if cart.adjustmentsTotal(promotionAdjustment) %}
+        {% set promotionTotal = cart.getPromotionsTotalRecursively() %}
```

Comment:
Nope, it [sums all promotions](https://github.com/Sylius/Sylius/pull/4347/files#diff-fb2a1dbd12848847780522681726f61dR530), which means this from order level and order item level as well.